target/linux/generic/pending-5.15/683-of_net-add-mac-address-to-of-tree.patch

```diff
@@ -45,11 +45,11 @@ property. This way, the MAC address can be accessed using procfs.
  /**
   * of_get_mac_address()
   * @np:		Caller's Device Node
-@@ -165,6 +186,7 @@ found:
- 		addr[5] = (mac_val >> 0) & 0xff;
- 	}
+@@ -140,6 +161,7 @@ int of_get_mac_address(struct device_nod
+ 	if (!ret)
+ 		return 0;
  
 +	of_add_mac_address(np, addr);
- 	return ret;
+ 	return of_get_mac_addr_nvmem(np, addr);
```

Comment:
I think the logic is wrong here. This may break the label mac hack. Suggested change: https://github.com/DragonBluep/openwrt/commit/c3a6146b4b4e59bdfb8e39abbb5861698c7da1d0

Reply:
Pulled.

Reply:
If you like, please keep your signature. Thanks for pushing it forward. It's really a hard job to convert all targets.

Reply:
sure.